src/test/java/com/easyrest/tests/BaseTest.java

```diff
@@ -70,7 +59,6 @@ public void tearDown() {
         driver.quit();
 
         //indicates that test is done and will not be monitored anymore
-        extent.flush();
```

Comment:
Where do you call flush()? I could not find this.

Reply:
I call flush in Listeners on finish method